<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskOneWidget</class>
 <widget class="QWidget" name="TaskOneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1769</width>
    <height>923</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1769</width>
    <height>923</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1769</width>
    <height>923</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>911</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="uploadRegexButton">
      <property name="text">
       <string>上传正则表达式文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uploadCodeButton">
      <property name="text">
       <string>上传源代码文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveFileButton">
      <property name="text">
       <string>保存当前文本框内容为文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="regexAnalyseButton">
      <property name="text">
       <string>正则表达式分析</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="codeAnalyseButton">
      <property name="text">
       <string>源代码词法分析</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>1191</width>
     <height>851</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTableView" name="NFAWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>801</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTableView" name="DFAWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>811</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QTableView" name="minDFAWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>811</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>811</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QTableView" name="codeAnalyseWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1181</width>
          <height>811</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>1191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请选择你要展示转换表的正则表达式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
